nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -330,7 +330,7 @@ geometry_msgs::msg::TwistStamped RegulatedPurePursuitController::computeVelocity
       costAtPose(pose.pose.position.x, pose.pose.position.y), linear_vel, sign);
 
     // Apply curvature to angular velocity after constraining linear velocity
-    angular_vel = linear_vel * curvature;
+    angular_vel = linear_vel * sign *curvature;
```

Comment:
```suggestion
    angular_vel = sign * linear_vel * curvature;
```

Fixes a linting problem and also a little more semantically clear if the `sign` is in front